src/modules/nfts/components/wizard/RoyaltiesCreators.tsx

```diff
@@ -0,0 +1,706 @@
+            <Form.Item
+              name="royaltiesPercentage"
+              rules={[
+                { required: true, message: 'Required' },
+                {
+                  type: 'number',
+                  min: 1, // Can this be 0? // we set it to 100 elsewhere, sooo?
+                  max: 100,
+                  message: 'Percentage must be between 1 and 100',
+                },
+              ]}
+              // initialValue={royaltiesInput}
+            >
+              <InputNumber<number>
+                min={1}
+                max={100}
+                formatter={(value) => `${value}%`}
+                parser={(value) => parseInt(value?.replace('%', '') ?? '0')}
+                style={{ borderRadius: 4, minWidth: 103 }}
+                onChange={(val: number) => setRoyaltiesInput(val)}
+              />
+            </Form.Item>
+          </Form.Item>
+          {/* Display creators */}
+          {creators.length < MAX_CREATOR_LIMIT && (
+            <Row justify="space-between" align="middle">
+              <Paragraph style={{ fontWeight: 900 }}>Creators split</Paragraph>
+              <StyledClearButton type="text" noStyle onClick={() => toggleCreatorField(true)}>
+                Add Creator
+              </StyledClearButton>
+            </Row>
+          )}
+          <Row>
+            {creators.map((creator) => (
+              <CreatorsRow
+                creatorAddress={creator.address}
+                share={creator.share}
+                isUser={creator.address === userKey}
+                key={creator.address}
+                updateCreator={updateCreator}
+                removeCreator={removeCreator}
+              />
+            ))}
+          </Row>
+          {/* Add creator form */}
+          {showCreatorField && (
+            <Row>
+              <Form.Item
+                name="addCreator"
+                style={{ width: '100%' }}
+                rules={[
+                  { required: true, message: 'Please enter a value' },
+                  { max: 44 },
+                  // need to handle someone pasting in a string longer than 44 char. Right now the input just caps it at 44 without showing anything
+                  { min: 44, message: 'Must be at least 44 characters long' },
+                  {
+                    message: 'All creator hashes must be unique',
+                    async validator(rule, value: string) {
+                      const existingCreators = creators.map((c) => c.address);
+                      const indexOfDuplicate = existingCreators.findIndex((a, i) => value === a);
+                      if (indexOfDuplicate !== -1) {
+                        throw new Error();
+                      }
+                    },
+                  },
+                  {
+                    message: 'Creator address is not valid',
+                    async validator(rule, creatorAddress: string) {
+                      if (creatorAddress.indexOf(' ') >= 0) {
```

Comment:
would it be possible to remove whitespace automatically rather than throwing a validation error?

Reply:
I suppose we could add on a trim and replace and just remove the rule entirely. What do you think? 
```
{ address: values.addCreator.trim().replace(/\s+/g,''), share: newShareSplit },
```

Reply:
That's what I was thinking, yeah.